src/constants/Constants.js

```diff
@@ -190,4 +195,9 @@ export default class Constants {
       label: "English",
     },
   };
+
+  static VALIDATION_SEVERITY = {
```

Comment:
Can we make ticket for spliting this file into model and rest ?

Reply:
Done here: https://github.com/kbss-cvut/s-forms/issues/279